src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const obj = {};
+
+  for (const key in robot) {
+    if (robot[key] in obj) {
+      return null;
+    }
+
+    obj[robot[key]] = key;
```

Comment:
You use robot[key] twice, so it's better to create a variable for that value, to make your code cleaner